form/src/main/java/org/dhis2/form/ui/FormView.kt

```diff
@@ -733,32 +732,35 @@ class FormView : Fragment() {
             .show()
     }
 
-    private fun showDescriptionLabelDialog() {
-        ComposeDialogFragment()
-            .show(childFragmentManager, "ComposeDialog")
-    }
+    private fun showDialog(intent: RecyclerViewUiEvents.ShowDescriptionLabelDialog) {
+        val dataElement = getD2().dataElement(intent.uid)
 
-    class ComposeDialogFragment : DialogFragment() {
-        override fun onCreateView(
-            inflater: LayoutInflater,
-            container: ViewGroup?,
-            savedInstanceState: Bundle?
-        ): View {
-            return ComposeView(requireContext()).apply {
-                setContent {
-                    MediaDialog(
-                        title = randomTitle(),
-                        subTitle = randomSubTitle(),
-                        mediaEntities = randomMediaEntities(),
-                        onDismiss = {
-                            dismiss()
-                        }
-                    )
-                }
-            }
+        if (dataElement.valueType()?.isFile == true) {
```

Comment:
Must check if the dataElement uid existes in dataStore, if yes, show the custom dialog else show the default dialog